src/test/datascience/.env

```diff
@@ -1,2 +1 @@
 ENV_VAR_TESTING_CI=HelloWorldEnvVariable
-PYTHONPATH=./dummyFolderForPythonPath
```

Comment:
@DonJayamanne is this expected to work? It looked like in the failures that it was only using this as the path. For a windows store python at least this breaks any test from running.

Reply:
Actually it's worse than I thought. Does more than break the test. It breaks everything just by existing. See this bug here:
https://github.com/microsoft/vscode-jupyter/issues/9127